kustomize/move2kube/base/config.properties

```diff
@@ -0,0 +1,4 @@
+MOVE2KUBE_URL=http://move2kube-svc.default.svc.cluster.local:8080
```

Comment:
the value should be the same as `MOVE2KUBE_API` (which is used by the knative function)
As the CM are shared between both the workflow and the knative function maybe it's worth  a refactoring of the move2kube to use the same name? (once this is merged) (and I'll do it)

And also, this URL is used in the notification as link to the transformation, may be better to put the route URL as default to make it obvious we want something reachable from the outside of the cluster

Reply:
These changes require a coordinate review of both repos. I suggest to postpone them once the current deployments are stabilized